tracing/tracing/src/main/java/io/helidon/tracing/NoOpTracer.java

```diff
@@ -136,6 +136,16 @@ public void end(Throwable t) {
         public io.helidon.tracing.Scope activate() {
             return SCOPE;
         }
+
+        @Override
+        public io.helidon.tracing.Span baggage(String key, String value) {
+            return null;
```

Comment:
You cannot return null. Return `this`

Reply:
It's NOT fixed! Please don't mark conversations as resolved if it's not fixed!